assets/scss/_base.scss

```diff
@@ -217,3 +217,13 @@ img.emoji {
   margin: 0 .05em 0 .1em;
   vertical-align: -0.1em;
 }
+
+#dark-mode-toggle {
+  bottom:0;
+  left:0;
+  z-index: 100;
+  position: fixed;
+  width:3em;
+  height:3em;
+  font-size: 1.6em;
```

Comment:
You can remove both `width` and `height` rules here. And set `bottom` and `left` to `2rem`.